makefile

```diff
@@ -16,7 +16,7 @@ ISVCS_NAME := serviced-isvcs
 ISVCS_VERSION     := 44-dev
 
 ZOOKEEPER_NAME := isvcs-zookeeper
-ZOOKEEPER_VERSION := 4-dev
+ZOOKEEPER_VERSION := 5-dev
```

Comment:
I don't think you need to bump this.  In the CC repo, we have
```
isvcs/isvc.go:	ZK_IMAGE_TAG  = "v3"
```
So leave the version here at "4-dev" until you're ready to release. Then change it to "4", build/publish the image, then change to "5-dev" in preparation for the next release. There's an outline of the process here - https://github.com/zenoss/zenoss-centos-base